src/model.rs

```diff
@@ -105,6 +105,63 @@ impl Pod {
     }
 }
 
+/// Struct to represent ``PodJob``
+#[derive(Serialize, Deserialize, Debug, PartialEq, Clone)]
+pub struct PodJob {
+    /// Optional annotation for pod job
+    pub annotation: Option<Annotation>,
+    /// Computed by coverting it to yaml then hash
+    pub hash: String,
+    /// Details about the pod from which the pod job was created from
+    pub pod: Pod,
+    input_volume_map: BTreeMap<PathBuf, PathBuf>,
```

Comment:
File hash -> stream key. Bring this up for dicussion tomrrow.
Perhaps need to implement input / store first.